python/mim_control/robot_centroidal_controller.py

```diff
@@ -70,7 +63,8 @@ def compute_com_wrench(self, q, dq, des_pos, des_vel, des_ori, des_angvel):
         vcom = np.reshape(np.array(dq[0:3]), (3,))
         Ib = robot.mass(q)[3:6, 3:6]
 
-        quat_diff = self.quaternion_difference(arr(q[3:7]), arr(des_ori))
+        tmp =  pin.Quaternion(des_ori[3], des_ori[0], des_ori[1], des_ori[2]) * pin.Quaternion(q[6], q[3], q[4], q[5]).conjugate()
```

Comment:
Does self.quaternion_difference work? If it doesn't work, please replace your code with the code in the function and here only call the function.